pkg/skuba/actions/node/join/join.go

```diff
@@ -68,6 +68,12 @@ func Join(joinConfiguration deployments.JoinConfiguration, target *deployments.T
 		"skuba-update.start",
 	}
 
+	if _, err := os.Stat(skuba.CriDockerDefaultsConfFile()); err == nil {
+		statesToApply = append(statesToApply, "")
+		copy(statesToApply[3:], statesToApply[2:])
```

Comment:
I don't like this hardcoded values. In case someone changes the order of states it might be a situation (by accident) where `crio.configure` will be called after the `crio.start`

Reply:
Fixed now.